### DISCUSSION TITLE
Adding the promotional videos

### DISCUSSION THREAD

**Discussion by kallepronk** [xl3ehindTim/delta-site#55] [Ideas]
About 2 years ago, a few delta's made new promotional material on youtube. We could show that on the about us as well.

**Comment by xl3ehindTim** [comment]
Love this @kallepronk!

**Comment by xl3ehindTim** [comment]
Are you referring to [this ](https://www.youtube.com/watch?v=_Kl2SxNfjA8&ab_channel=DeltaFHICT)video @kallepronk?
